src/ansiblelint/rules/__init__.py

```diff
@@ -107,24 +108,17 @@ def matchtasks(self, file: Lintable) -> List[MatchError]:
         ):
             return matches
 
-        yaml = ansiblelint.utils.parse_yaml_linenumbers(file)
-        if not yaml:
-            return matches
-
-        yaml = ansiblelint.skip_utils.append_skipped_rules(yaml, file)
-
-        try:
-            tasks = ansiblelint.utils.get_normalized_tasks(yaml, file)
```

Comment:
After removing this line, `get_normalized_tasks` is no longer used anywhere. That means that code coverage decreases because nothing is using it. Should we deprecate or remove `get_normalized_tasks`? Or perhaps add a test for `get_normalized_tasks` in case any 3rd party rules are using it (not sure why they would)?

Reply:
I removed `get_normalized_tasks`. If we need to deprecate it or add a test instead, then let me know and I can revert the delete commit.

Reply:
Oh. But removing it still decreases overall coverage (albeit slightly less).

Reply:
Oh. never mind. I was looking at an old codecov report. Codecov is green now.